suites/nautilus/cephfs/tier-2/snapshot_clone.yaml

```diff
@@ -0,0 +1,75 @@
+tests:
```

Comment:
Could we have the file name as `tier-2_cephfs_test-snapshot-clone.yaml`? This allows us to retrieve the metadata information from the filename.

Reply:
Done